src/desktop/components/main_layout/header/templates/more.jade

```diff
@@ -10,6 +10,3 @@ span.mlh-top-nav-link.hover-pulldown( data-mode='hover' )
       a.mlh-pulldown-top-link-persistent( href='/artists', class= activeClass('/artists') ) Artists
       a.mlh-pulldown-top-link-persistent( href='/shows', class= activeClass('/shows') ) Shows
       a.mlh-pulldown-top-link-persistent( href='/institutions', class= activeClass('/institutions') ) Museums
-    hr.mlh-pulldown-top
-    div.mlh-pulldown-bottom
-      a.mlh-pulldown-bottom-link( href='/about' ) About Artsy
```

Comment:
These actually had no associated styles, so it was just a matter of deleting the markup.